app/models/item.rb

```diff
@@ -4,5 +4,5 @@ class Item < ApplicationRecord
 
   has_and_belongs_to_many :auth_ip_addresses
   has_and_belongs_to_many :auth_services
-  belongs_to :end_service
+  belongs_to :end_service, optional: true
```

Comment:
Hi @pachonk ,

@ekate is looking at the whole PR. But I had a question about this line. Could you tell me why belongs_to end_service is optional here? Are you thinking there would be a use case where an item doesn't belong to an end service?  Thanks.